base/abstractarray.jl

```diff
@@ -38,18 +38,32 @@ function first(itr)
 end
 last(a) = a[end]
 
+"""
+    stride(A, k)
+
+Returns the distance in memory (in number of elements) between adjacent elements in dimension `k`.
+"""
 function stride(a::AbstractArray, i::Integer)
-    if i > ndims(a)
-        return length(a)
-    end
+    i > ndims(a) && return length(a)
```

Comment:
I don't think we should actively change code to a format that generates worse coverage specificity.